Client/Frontend/Settings/Clearables.swift

```diff
@@ -41,15 +41,19 @@ class HistoryClearable: Clearable {
     }
 
     func clear() -> Success {
-        return profile.history.clearHistory().bind { success in
```

Comment:
curuious why we cant use `bindQueue(.main)`

Reply:
I'll switch that,  agreed ( that here `return bindQueue(.main)` is equivalent to `uponQueue(.main) { fill deferred }; return deferred`). I recall not finding any use of bind() in the Client module so I thought upon/uponQueue was the prevailing style.